packages/app-page-builder/src/editor/components/Element.tsx

```diff
@@ -101,29 +136,31 @@ const Element = (props: ElementProps) => {
     }
 
     return (
-        <Transition in={true} timeout={250} appear={true}>
-            {state => (
-                <ElementContainer
-                    id={element.id}
-                    onMouseOver={onMouseOver}
-                    highlight={highlight}
-                    active={active}
-                    style={{ ...defaultStyle, ...transitionStyles[state] }}
-                    className={"webiny-pb-page-element-container"}
-                >
-                    <div className={["innerWrapper", className].filter(c => c).join(" ")}>
-                        <Draggable target={plugin.target} beginDrag={beginDrag} endDrag={endDrag}>
-                            {renderDraggable}
-                        </Draggable>
-                        {plugin.render({ element })}
-                    </div>
-                    {/*
-                        <div className="add-element add-element--above">+</div>
-                        <div className="add-element add-element--below">+</div>
-                        */}
-                </ElementContainer>
-            )}
-        </Transition>
+        <ErrorBoundary>
+            <Transition in={true} timeout={250} appear={true}>
+                {state => (
+                    <ElementContainer
+                        id={element.id}
+                        onMouseOver={onMouseOver}
+                        highlight={highlight}
+                        active={active}
+                        style={{ ...defaultStyle, ...transitionStyles[state] }}
+                        className={"webiny-pb-page-element-container"}
+                    >
+                        <div className={["innerWrapper", className].filter(c => c).join(" ")}>
+                            <Draggable target={plugin.target} beginDrag={beginDrag} endDrag={endDrag}>
+                                {renderDraggable}
+                            </Draggable>
+                            {plugin.render({ element })}
```

Comment:
I think we only want to wrap this line with the ErrorBoundary. If an error occurs, we still want to be able to select and delete the element.